packages/react-dev-utils/launchEditor.js

```diff
@@ -176,6 +187,17 @@ function guessEditor() {
           return [fullProcessPath];
         }
       }
+    } else if (process.platform === 'linux') {
+      const output = child_process
+        .execSync('ps --no-heading -e -o comm --sort=comm')
+        .toString();
+      const processNames = Object.keys(COMMON_EDITORS_LINUX);
+      for (let i = 0; i < processNames.length; i++) {
+        const processName = processNames[i];
+        if (output.indexOf(processName) !== -1) {
```

Comment:
Should we worry about a process which is named like, `mycode` or something matching `code`?
Can we make sure it's delimited/a whole word?

Reply:
We'd probably want to apply this logic to macOS, too.
Not sure if you have a macOS box so we could do this in a separate PR.

Reply:
I`am not sure.```ps -o comm``` output looks nice. It takes 
```
The name of the command being executed ( argv[0] value) as a string. 
```
and returns something like: 
```
kworker/u8:3
led_workqueue
lightdm
```

Reply:
We can worry about it later. 😄